linux/home/radar/ros.3.6/usr/codebase/superdarn/src.lib/os/site.fhw.1.0/src/site.c

```diff
@@ -1076,14 +1076,14 @@ usleep(usecs);
         if (debug)
         fprintf(stderr,"FHW seq %d :: ACFCalculate acf\n",nave);
         ACFCalculate(&tsgprm,(int16 *) dest,rngoff,skpnum!=0,
-          roff,ioff,mplgs,lagtable,acfd,ACF_PART,2*total_samples,badrng,seqatten[nave]*atstp,NULL);
+          roff,ioff,mplgs,lagtable,acfd,ACF_PART,2*dprms.samples,badrng,seqatten[nave]*atstp,NULL);
```

Comment:
Still looking back through my emails to see if this is the correct change, but I believe there is a typo here - should be `dprm.samples` and not `dprms.samples`

Reply:
Whoops.  I'd found that while trying to compile but forgot to push it up to git.